src/integration-test/java/org/openlmis/requisition/repository/RequisitionRepositoryIntegrationTest.java

```diff
@@ -233,6 +235,19 @@ public void testSearchRequisitionsByTemplate() {
   }
 
   @Test
+  public void shouldFindRequisitionsByMultipleIds() {
+    UUID id1 = requisitions.get(0).getId();
+    UUID id2 = requisitions.get(1).getId();
+
+    List<Requisition> foundRequisitions = Lists.newArrayList(repository.findAll(
```

Comment:
Not absolutely necessary, but findAll returns an object of Iterable, so I'm doing this for convenience.